migrate_repo/versions/008_Add_shipped_at.py

```diff
@@ -0,0 +1,25 @@
+# Upgrade/downgrade the database with the shippedAt field.
+# CF bug #1101596 for more information
+from sqlalchemy import Column, MetaData, Table, DateTime
+
+import pytz
+
+from datetime import datetime
+
+
+def upgrade(migrate_engine):
+    metadata = MetaData(bind=migrate_engine)
+
+    def add_shippedAt(table):
+        shippedAt = Column('shippedAt', DateTime(pytz.utc))
+        shippedAt.create(table)
```

Comment:
What are we going to do about backfilling old release ship dates? Do we need to?